test/parallel/test-fs-open-flags.js

```diff
@@ -22,6 +23,8 @@ assert.strictEqual(stringToFlags('w+'), O_TRUNC | O_CREAT | O_RDWR);
 assert.strictEqual(stringToFlags('a'), O_APPEND | O_CREAT | O_WRONLY);
 assert.strictEqual(stringToFlags('a+'), O_APPEND | O_CREAT | O_RDWR);
 
+assert.strictEqual(stringToFlags('rs+'), O_RDWR | O_SYNC);
```

Comment:
Updated.

I'll update a doc:) Maybe it is [here](https://github.com/nodejs/node/blob/master/doc/api/fs.md#fsopenpath-flags-mode-callback)?